README.md

```diff
@@ -0,0 +1,40 @@
+# keeper
+
+## Setup
+
+This is how to get keeper running on your machine so that you're able to work on it.
+
+### 1. Clone the repo and install node modules
+```
+git clone git@github.com:geckoboard/keeper.git
+cd keeper
+yarn
+```
+
+### 2. Install postgresql and create the keeper database
+```
+brew install postgresql
+createdb keeper-dev
+```
+
+If creating the db gives you an error, postgresql might not be running. To start it run this:
+```
+brew services start postgresql
+```
+
+### 3. Update the keeper config file
+In `server/server/config/config.json`, change the username `dan` to the username on your machine. If you don't know what it is, `whoami` will give it to you.
```

Comment:
The username can probably be an environment variable later.

Reply:
yeah good point 🤔